src/App.js

```diff
@@ -1,17 +1,21 @@
 import React from 'react';
 
+const todoList = [
+  { id: 1, title: 'Complete assigment' },
+  { id: 2, title: 'Play with kids' },
+  { id: 3, title: 'Watch TV' },
+];
+
 function App() {
   return (
-    <div style={{ textAlign: 'center' }}>
-      <header>
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a href="https://reactjs.org" target="_blank" rel="noopener noreferrer">
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+      <h1>Todo list</h1>
+      <ul>
+        {todoList.map((item) => {
+          return <li key={item.id}>{item.title}</li>;
+        })}
+      </ul>
+    </>
```

Comment:
nice job! I like what I see here.
Could you please explain to me what this line of code is doing `<li key={item.id}>{item.title}</li>`. Why am I not seeing the key for a given item in the list?

Reply:
Thank you. The line of code is part of a React component, which dynamically renders a list item (<li> element). You aren't seeing the key when inspecting the rendered HTML because React uses the key attribute internally and does not include it as a visible attribute in the DOM.

Reply:
Well done! Keep it up!